cheatsheets/PHP_Configuration_Cheat_Sheet.md

```diff
@@ -111,3 +107,13 @@ report_memleaks         = On
 track_errors            = Off
 html_errors             = Off
 ```
+
+## Suhosin (PHP 5.X)
+
+Consider using [Suhosin](https://suhosin.org/) if you want to patch many custom security flaws in various parts of PHP. 
+Note: only usable in 5.X branch and should not be used in prod now as 5.X is EOL.
```

Comment:
Beware of markdown new line. Line 113 and line 114 are on the same line in markdown format.

I'd recommend the following as well:
```suggestion
> **Suhosin is only usable on the 5.X branch and should not be used on production servers as 5.X reached EOL.**
```